### PR TITLE
Assignment 3 completed

### PR DESCRIPTION
**Pull request by Shreya1304**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

**Comment by aravindvnair99** [comment]
@Shreya1304 Next time fill up the pull request template properly.

File: Shreya/HTML Basics Remake/form.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<html>
+    <body>
+
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html]: Missing head tag

File: Shreya/HTML Basics Remake/form.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<html>
+    <body>
+        
+        <form>
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html]: Missing identifier for the form

**Comment by Shreya1304** [comment]
Sorry, I didn't get you

File: Shreya/HTML Basics Remake/form.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                <input type="text" name="last" placeholder="Last Name" required>
+           </p>
+            <p>
+                <label for="gender">Gender:</label>
+                <label for="male">Male:</label>
+                <input type="radio" id="male" required>
+                <label for="female">Female:</label>
+                <input type="radio" id="female" required>
+                <label for="others">Others:</label>
+                <input type="radio" id="others" required>
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html]: Radio buttons are not present in one group thus leading to the selection of multiple.

File: Shreya/HTML Basics Remake/form.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            </p>
+            <p>
+                <label for="email">Email:</label>
+                <input type="email" placeholder="Email">
+                <label for="password">Password:</label>
+                <input type="password" pattern=".{5,15}" title="PLease enter minimum 5 and maximum 15 characters">
+                <label for="date">Birthday</label>
+            <input type="date">
```
Comment (aravindvnair99) [inline Shreya/HTML Basics Remake/form.html]: Missing required attribute

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@Shreya1304 Fix the indentation to tab 4.

**Comment by aravindvnair99** [comment]
> Sorry, I didn't get you

You need to fill up the pull request template. Not just submit as it is.

**Comment by Shreya1304** [comment]
Done

**Comment by aravindvnair99** [comment]
@Shreya1304 Did you close the pull request accidentally?

**Comment by Shreya1304** [comment]
Oops , yes , what shall I do now?

**Comment by Shreya1304** [comment]
I sent you a new one again, with the pull request template.

**Comment by aravindvnair99** [comment]
> Oops , yes , what shall I do now?
> I sent you a new one again, with the pull request template.

@Shreya1304 You can always reopen a closed pull request. Don't have to make a new one. And what you type can be edited, so the pull request description can always be edited.

**Pull request closed without merging**